survey/templates/survey/survey_myplanner_content.html

```diff
@@ -17,12 +17,12 @@
                                         <table class="layer-row">
                                             <tbody>
                                                 <tr>
-                                                    <td>
+                                                    <!-- <td> -->
                                                         <!-- <a class="btn btn-info-sign" data-bind="click: toggleDescription, css: { 'active': infoActive()}" data-toggle="tooltip" data-placement="right" data-title="More Info" data-original-title="" title=""> -->
-                                                        <a class="btn btn-info-sign"  data-toggle="tooltip" data-placement="right" data-title="More Info" data-original-title="" title="">
+                                                        <!-- <a class="btn btn-info-sign"  data-toggle="tooltip" data-placement="right" data-title="More Info" data-original-title="" title="">
```

Comment:
what is the reason for commenting these out?

Reply:
Totally, makes sense to me. I could see a TODO comment noting when they should be commented back in helpful if we were going to leave them uncommented for a while. But since you said the quality-of-life features will be added shortly it might not be necessary 🤷🏻